codice/alliance
# CAL-116 Added MGMP Integration test, documentation, mock server

#### What does this PR do?
This PR adds a mock MGMP server for testing, adds an integration test, and updates the documentation for Alliance.
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@glenhein @jrnorth 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@jlcsmith
@kcwire
#### How should this be tested?
Build and Install Alliance Experimental, run the MGMP mock server (/distribution/sdk/sample-mgmp-server) and configure the MGMP source (http://localhost:9081/csw) and observe appropriate behavior.
#### Any background context you want to provide?
This PR is built on top of https://github.com/codice/alliance/pull/110
#### What are the relevant tickets?
https://codice.atlassian.net/browse/CAL-116
#### Screenshots (if appropriate)
#### Checklist:
